### PR TITLE
Medicine

### PR DESCRIPTION
**Pull request by Bjoern**
**What does your Instant Answer do?**
Provide details about medicine

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Medicine can be quickly identified and detailed information can be provided

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://myHealthbox.eu

**Why did you choose this data source?**
Promotion of myHealthbox.eu

**Are there any other alternative (better) data sources?**

**What are some example queries that trigger this Instant Answer?**
aspirin dosage

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
patients

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
https://duck.co/ideas/idea/4607/looking-for-developer-to-help-with-ddg-spice-insta

**Which existing Instant Answers will this one supersede/overlap with?**

**Are you having any problems? Do you need our help with anything?**
Yes, not sure how make a detail snippet for the leaflet information.

**What are the terms of use for the API? Will DuckDuckGo need specific authorization (e.g. an API key)? Are there any costs associated with API usage?**
No costs

**Where did you hear about DuckDuckHack? (For first time contributors)**

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
Just a draft so far:
myhealthboxddg

## Checklist
Please place an 'X' where appropriate.

```
[] Added metadata and attribution information
[X] Wrote test file and added to t/ directory
[] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [] Google Chrome
        [] Firefox
        [] Opera
        [] Safari

    - iOS (iPhone)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [] Native Browser
        [] Google Chrome
        [] Opera
```

### CODE REVIEW

**Comment by MrChrisW** [comment]
@bjoern Looks like you've accidentally included some of your other code in this PR? Did you mean to? 

I'll try my best to help with the template issues when I get a free moment :+1:

**Comment by Bjoern** [comment]
Hi,

yeah, sorry. I guess the problem is that I had developed my first IA in
the master branch (UV Index), then I branched medicine from that. Or
maybe I shouldn't have merged upstream/master into the branch :-/

My git fu is not strong enough to fix that immediately, but I can look
into it. Always good to learn more about git...

I made the pull request because I was asked to do so on the mailing
list, to make it easier to discuss problems. It is not finished yet.

Björn

**Comment by MrChrisW** [comment]
Ok, after a little research I've usually found it easier to use the "products" template group and use the `buy` option in `options:` to specify the handlebars template to display tile content. 

However @moollaza probably knows a better way (best practice) :smile:

**Comment by MrChrisW** [comment]
@bjoern That's fine! I thought that was the case! We can still review the code anyway. You can simply do a `git rm <file>` and remove the UV files. Then commit and push (in this branch)

**Comment by Bjoern** [comment]
I'll try the product group. Meanwhile, here is a screenshot of the Q/A instant answer, which looks like a good fit. But I couldn't find the code (it's not the maven IA I think, something else was triggered here).
ddg_qa

Or is there perhaps a way to override styles of existing templates? I think the 'text' group would work well if the titles wouldn't be cut off.

File: share/spice/github/github.js [hunk 1]
```diff
@@ -11,7 +11,7 @@
                     .replace(/^\s*github\s+/, "");
 
         var results = api_result.data.repositories;
-
+
```
Comment (moollaza) [inline share/spice/github/github.js]: Looks like this also needs to be removed from the PR?

**Comment by moollaza** [comment]
@Bjoern unfortunately the StackOverflow Instant Answer is closed-source which is why you can't find the code.

As you mentioned, the `text` template group is ideal for this kind of data. You're also able to provide a CSS file that can be used to modify the UI if necessary. I'd say go ahead with the `text` template group for now so we can see how it looks. If we need to make any adjustments our designers will be able to tell you what needs to be adjusted.

**Comment by Bjoern** [comment]
I managed to correct the title via CSS (smaller font, no cropping), but is there also a way to get linebreaks into the description of a text_item? Thanks!

**Comment by moollaza** [comment]
Hi @Bjoern,

Thanks a lot for making this, we really appreciate you taking the time to contribute.

After discussing this PR with the team, playing around with the Spice and investigating the API a little more, we've decided that MyHealthBox isn't a good fit for what we want out of a medicine/dosage Instant Answer.

Right now the API doesn't give very much info (only about 5 data points) and there's no good place to direct users for more info because MyHealthBox requires a signup and just links to a very long PDF, so unfortunately I don't see this becoming a very helpful Instant Answer.

I'm very sorry that your time and effort has been wasted, but I hope you can understand our reasoning for closing this.

**Pull request closed without merging**